v03_pipeline/lib/annotations/sv.py

```diff
@@ -81,6 +81,41 @@ def _sv_types(ht: hl.Table) -> hl.ArrayExpression:
     return ht.alleles[1].replace('[<>]', '').split(':', 2)
 
 
+def alleles(ht: hl.Table, **_: Any) -> hl.ArrayExpression:
+    return hl.array(
+        [
+            'N',
+            hl.if_else(
+                (
+                    hl.is_defined(ht.sv_type_detail_id)
+                    & (hl.array(SV_TYPES)[ht.sv_type_id] != 'CPX')
+                ),
+                hl.format(
+                    '<%s:%s>',
+                    hl.array(SV_TYPES)[ht.sv_type_id],
+                    hl.array(SV_TYPE_DETAILS)[ht.sv_type_detail_id],
+                ),
+                hl.format('<%s>', hl.array(SV_TYPES)[ht.sv_type_id]),
+            ),
+        ],
+    )
+
+
+def info(ht: hl.Table, **_: Any) -> hl.StructExpression:
+    return hl.Struct(
+        ALGORITHMS=ht.algorithms,
+        END=ht.start_locus.position,
+        CHR2=ht.end_locus.contig,
+        END2=ht.end_locus.position,
+        SVTYPE=hl.array(SV_TYPES)[ht.sv_type_id],
+        SVLEN=ht.sv_len,
+    )
+
+
+def locus(ht: hl.Table, **_: Any) -> hl.LocusExpression:
```

Comment:
Is the plan to keep both `locus` and `start_locus`, or to rename the field?

Reply:
Keep only `start_locus` for our annotations table, then export to `locus`, `alleles`, `info`